extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -112,6 +114,14 @@ export class AmpSlideScroll extends BaseSlides {
 
     /** @private {?../../../src/service/action-impl.ActionService} */
     this.action_ = null;
+
+    /** @private {boolean} */
+    this.isIosBetaExperimentOn_ = isExperimentOn(this.win,
+        'slidescroll-ios-beta');
+
+    /** @private {boolean} */
+    this.isIosBeta_ = startsWith(platformFor(this.win).getIosVersionString(),
+        '10.3');
```

Comment:
Right now the 10.3 is the only problematic version, I will create a followup issue for comparing IOS minor version.